test/e2e/antreaipam_test.go

```diff
@@ -1,7 +1,43 @@
 package e2e
 
 import (
+	"bytes"
+	"context"
+	"fmt"
+	"net"
 	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	crdv1alpha2 "antrea.io/antrea/pkg/apis/crd/v1alpha2"
+)
+
+const (
+	// TODO(gran): correct prefix
+	IPPoolAnnotationKey string = "ipam.antrea.io/ippools"
```

Comment:
Why don't import the variable from `pkg`?

Reply:
Thanks. Changed to const in ipam pkg. This code is earlier than Anna's patch, thus I used a const.